osx/KBKit/KBKit/Component/KBCommandLine.m

```diff
@@ -58,6 +53,7 @@ - (void)uninstall:(KBCompletion)completion {
 }
 
 - (void)refreshComponent:(KBRefreshComponentCompletion)completion {
+  // Also consider us installed if there is a file in /usr/local/bin/keybase
```

Comment:
I fixed `installCommandLine` in `go/install` package to overwrite /usr/local/bin/keybase file if it doesn't resolve to the right place or isn't a link at all.

I also updated the privileged command line install (via KBHelper), to remove any existing path before install. So it does the same thing (but privileged).

This means that in the case where /usr/local/bin/keybase exists as a binary it will be overritten with correct link.

We didn't have a problem before because brew does symlinks in /usr/local/bin and macports uses /opt/ by default.

Updating the build number in helper since it's not worth upgrading existing people.